test/test-WebRtcTransport.js

```diff
@@ -96,15 +96,12 @@ test('router.createWebRtcTransport() succeeds', async () =>
 	expect(transport1.iceParameters.iceLite).toBe(true);
 	expect(transport1.iceParameters.usernameFragment).toBeType('string');
 	expect(transport1.iceParameters.password).toBeType('string');
-	expect(transport1.sctpParameters).toStrictEqual(
+	expect(transport1.sctpParameters).toMatchObject(
 		{
-			port               : 5000,
-			OS                 : 2048,
-			MIS                : 2048,
-			maxMessageSize     : 1000000,
-			isDataChannel      : true,
-			sctpBufferedAmount : 0,
-			sendBufferSize     : 262144
+			port           : 5000,
+			OS             : 2048,
+			MIS            : 2048,
+			maxMessageSize : 1000000
```

Comment:
Same here, why so many fields removed?

Reply:
This property represents https://mediasoup.org/documentation/v3/mediasoup/sctp-parameters/#SctpParameters struct, which doesn't have fields `isDataChannel`, `sctpBufferedAmount` and `sendBufferSize`. Identical Rust version of the test again didn't compile due to type mismatch.

These were again only used in tests and never described in public API, so I decided to clean that up and remove both from tests and worker itself for consistency.

If those properties are important to check, they need to be extracted from dump or elsewhere, not placed into `SctpParameters` where they don't belong.